tests/test_pipelines/test_mesh_pipelines.py

```diff
@@ -167,7 +167,7 @@ def test_mesh_pipeline():
     collect = Collect(
         keys=[
             'img', 'joints_2d', 'joints_2d_visible', 'joints_3d',
-            'joints_3d_visible', 'pose', 'beta', 'part_index', 'uv_coordinates'
+            'joints_3d_visible', 'pose', 'beta', 'index_mask', 'uv_coordinates'
```

Comment:
part_index_mask ?

Reply:
no, i mean part_index is clearer than index_mask

Reply:
OK, I will change the name back to part_index.